src/config.rs

```diff
@@ -14,6 +47,7 @@ pub struct Config {
     pub readyset_hostgroup: u16,
     pub warmup_time: Option<u16>,
     pub lock_file: Option<String>,
+    pub operation_mode: Option<OperationMode>,
```

Comment:
Do you need the `Option` here especially since you declare a default for the enum?

Reply:
I am using `toml` to parse the file into the struct. It requires the Option tag otherwise it will complain it cannot parse it.